src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] for the input values: (14, 14)', () => {
```

Comment:
can u give more information description about tests :  'should return 0 for cat and dog when they are 14 years old' as an example. Change for other tests

Reply:
Done